ls1intum/Artemis
# `Development`: Replace remaining uses of deprecated RxJS methods

### Checklist
#### General


#### Client

### Motivation and Context
The `subscribe` method with multiple parameters is deprecated and will be removed in RxJS 8.

### Description
Replaced their usages with [the new recommended format](https://rxjs.dev/deprecations/subscribe-arguments).

### Steps for Testing
Code review should be enough as only the mentioned methods have been changed to follow the new format. If no errors were made during the replacement, this results in no functional changes.
Setting the diff view to split instead of unified if reviewing on Github definitely makes the differences easier to see in this case.

There might be some other annotations about deprecated RxJS methods. Those have been or will be resolved when #4606 is merged. Together these two PRs replace all currently used deprecated methods from RxJS.

### Review Progress
#### Code Review

### Test Coverage
unchanged
